rules/dotnet-upgrade-for-complex-projects/rule.md

```diff
@@ -99,61 +99,74 @@ There are several ways to migrate project from ASP.NET to ASP.NET Core. We stron
 
 ### Create side-by-side incremental project with [.NET Upgrade Assistant](https://dotnet.microsoft.com/en-us/platform/upgrade-assistant)
 
-This will create a new .NET 8 project. For functionalities that have not yet been migrated, YARP will redirect them to the .NET Framework web application.
+After you've [installed the .NET Upgrade Assistant extension](https://learn.microsoft.com/en-au/dotnet/core/porting/upgrade-assistant-install#install-the-visual-studio-extension),
+right-click on the project in the Solution Explorer window, and select Upgrade.
 
-### Configure YARP
+![image](https://github.com/SSWConsulting/SSW.Rules.Content/assets/3699937/3303daaf-0dea-4b34-9f59-53fd55acf2ef)
+
+A tab is opened which provides, based on your project type, different styles of upgrade:
+
+- In-place project upgrade
+  This option upgrades your project without making a copy.
+
+- Side-by-side project upgrade
+  Copies your project and upgrades the copy, leaving your original project alone.
+
+- Side-by-side incremental
+  A good choice for complicated web apps. Upgrading from ASP.NET to ASP.NET Core requires quite a bit of work and at times manual refactoring. This mode puts a .NET project next to your existing .NET Framework project, and routes endpoints that are implemented in the .NET project, while all other calls are sent to .NET Framework application.

```

Comment:
I am not sure what is meant by the final sentence of this paragraph.

Reply:
@ncn-ssw 

> routes endpoints implemented in the .NET project, while all other calls are forwarded to the legacy .NET Framework application.

is that clearer? In simple English, if the request is implemented in the new project than it will be handled there otherwise it forwards the request to the original, old, legacy project, for it to handle.

Reply:
I don't believe it is. I recommend:

```suggestion
  A good choice for complicated web apps. Upgrading from ASP.NET to ASP.NET Core requires quite a bit of work and at times manual refactoring. This mode puts a .NET project next to your existing .NET Framework project.  If an endpoint is implemented in the new project, any requests to that endpoint will be handled in the new project. All other requests will be handled by the old project.
```